### PR TITLE
docs: Add instructions for bun/bunx

### PR DESCRIPTION
**Pull request by colinhacks**

### CODE REVIEW

**Comment by colinhacks** [comment]
@microsoft-github-policy-service agree [company="Bun"]

File: docs/src/intro-js.md [hunk 1]
```diff
@@ -47,6 +48,14 @@ yarn create playwright
 pnpm create playwright
 ```
 
+</TabItem>
+
+<TabItem value="bun">
+
+```bash
+bunx create-playwright
```
Comment (mxschmitt) [inline docs/src/intro-js.md:56]: `bunx` was not available, so I tried with `bun x` which wasn't working for me:

image

Comment (colinhacks) [reply docs/src/intro-js.md:56]: Apologies, how are you running this? Are you on a recent version of Bun? If `bunx` isn't available it's possibly quite old.

I see a different error (which I can also replicate with `npx`).

```sh
$ bunx create-playwright
Getting started with writing end-to-end tests with Playwright:
Initializing project in '.'
✔ Where to put your end-to-end tests? · playwright
✔ Add a GitHub Actions workflow? (y/N) · false
✔ Install Playwright browsers (can be done manually via 'npx playwright install')? (Y/n) · true
Installing Playwright Test (npm install --save-dev @playwright/test)…
npm ERR! Invalid Version: 

npm ERR! A complete log of this run can be found in: /Users/colinmcd94/.npm/_logs/2023-08-15T00_10_16_493Z-debug-0.log
Error: Command failed: npm install --save-dev @playwright/test
    at checkExecSyncError (node:child_process:887:11)
    at execSync (node:child_process:959:15)
    at executeCommands (/private/tmp/create-playwright@latest--bunx/node_modules/create-playwright/lib/index.js:4519:39)
    at Generator.run (/private/tmp/create-playwright@latest--bunx/node_modules/create-playwright/lib/index.js:4679:5)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async /private/tmp/create-playwright@latest--bunx/node_modules/create-playwright/lib/index.js:4952:3 {
  status: 1,
  signal: null,
  output: [ null, null, null ],
  pid: 12934,
  stdout: null,
  stderr: null
}
error: "create-playwright" exited with code 1 (SIGHUP)
```

Comment (mxschmitt) [reply docs/src/intro-js.md:56]: I used the latest bun available on bun.sh 0.7.3

Comment (colinhacks) [reply docs/src/intro-js.md:56]: I'm able to replicate your error when I do `bunx --bun create-playwright` which runs the create-playwright CLI with the Bun runtime. Opened an issue here: https://github.com/oven-sh/bun/issues/4153

Though without the `--bun` flag this command should respect the shebang line and run the CLI with Node.js. Maybe you have a global bunfig.toml that changes this behavior? 

I'm able to run `bunx create-playwright` successfully on my machine now...I think that "Invalid Version" error was a transient npm registry issue.

Comment (mxschmitt) [reply docs/src/intro-js.md:56]: It was a fresh installation from bun.sh on GitHub Codespaces, so no global bunfig.toml was present.

Generally speaking we should try to run our testsuite "npm run ctest" with bun and see how many tests pass/are failing.

**Comment by pavelfeldman** [comment]
Let's wait for the users to request interop with bun. We don't know if it works, so documenting it would be potentially setting up users for failure.

**Comment by infrahead** [comment]
@pavelfeldman FYI, it seems like Bun runtime support is close, but now stuck on this issue with imports/exports - https://github.com/oven-sh/bun/issues/2492#issuecomment-1645744477

**Comment by mxschmitt** [comment]
Without adding bots / testing infrastructure to make sure all the Playwright tests pass on bun, we are not confident enough to recommend this to our users.

**Pull request closed without merging**